pol-core/poltool/PolToolMain.cpp

```diff
@@ -42,34 +43,41 @@ void PolToolMain::showHelp()
               << "            x1 y1 [x2 y2 realm]       writes polmap info to polmap.html\n"
```

Comment:
was this already out of date?

Reply:
Do you mean due to the argument index change?  binargs contains the complete argc thus also the binary name which wasn't considered correctly.

Reply:
Ah yes because mapdump is [1]

Reply:
If I remember it long enough, we should fix it. That the vector programargs contains also the binary name isn't something I would expect. ;)